src/main/java/com/softserve/model/Category.java

```diff
@@ -0,0 +1,43 @@
+package com.softserve.model;
+
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Category {
+
+	@Id
+	@Column(name = "id_category")
+	private Integer idCategory;
+	
+	@Column(name = "\"name\"", length = 30)
```

Comment:
is the column name on DB as "name", including quotation marks? or why are you using the back slash before the quotation marks?

Reply:
We use like this because the word name is a keyword for postgres and we need to pass it like that.

Reply:
got it, thanks for the note